sdk/core/core/src/az_http_policy_retry.c

```diff
@@ -61,7 +61,9 @@ AZ_INLINE void _az_http_policy_retry_log(int16_t attempt, int32_t delay_msec)
   uint8_t log_msg_buf[AZ_LOG_MSG_BUF_SIZE] = { 0 };
   az_span log_msg = AZ_SPAN_FROM_BUFFER(log_msg_buf);
   (void)_az_http_policy_retry_append_http_retry_msg(attempt, delay_msec, &log_msg);
-  az_log_write(AZ_LOG_HTTP_RETRY, log_msg);
+
+  // The span is going to b xero terminated, we wanto to exclude that from the length.
```

Comment:
typos: be 0-terminated.
But, see my comments earlier

Reply:
Done. (N/A)